src/Parallel/Parallel.jl

```diff
@@ -1,19 +1,20 @@
 module Parallel
 
 using LightGraphs
-using LightGraphs: sample, AbstractPathState, JohnsonState, BellmanFordState, FloydWarshallState
+using LightGraphs: sample, AbstractPathState
+using LightGraphs.ShortestPaths
 using Distributed: @distributed
 using Base.Threads: @threads, nthreads, Atomic, atomic_add!, atomic_cas!
+using LightGraphs.ShortestPaths: NegativeCycleError
 using SharedArrays: SharedMatrix, SharedVector, sdata
 using ArnoldiMethod
+import Random
 using Random:shuffle
 import SparseArrays: sparse
 import Base: push!, popfirst!, isempty, getindex
+import LightGraphs.ShortestPaths: shortest_paths
```

Comment:
Maybe we can put this together with line 8 (and put it next to line 5)

Reply:
I think I fixed this in the latest commit.